script/tool/lib/src/common/plugin_command.dart

```diff
@@ -432,17 +434,48 @@ abstract class PluginCommand extends Command<void> {
     return gitVersionFinder;
   }
 
-  // Returns packages that have been changed given a list of changed files.
+  // Returns the names of packages that have been changed given a list of
+  // changed files.
+  //
+  // The names will either be the actual package names, or potentially
+  // group/name specifiers (for example, path_provider/path_provider) for
+  // packages in federated plugins.
   //
   // The paths must use POSIX separators (e.g., as provided by git output).
-  Set<String> _getChangedPackages(List<String> changedFiles) {
+  Set<String> _getChangedPackageNames(List<String> changedFiles) {
     final Set<String> packages = <String>{};
+
+    // A helper function that returns true if candidateName looks like an
+    // implementation package of a plugin called pluginName. Used to determine
+    // if .../packages/parentName/candidatePackageName/...
+    // looks like a path in a federated plugin package (candidatePackageName)
+    // rather than a top-level package (parentName).
+    bool isFederatedPackage(String candidatePackageName, String parentName) {
```

Comment:
s/isFederatedPackage/_isFederatedPackage

Reply:
Locals shouldn't be named with underscores.